src/sharding/ShardingManager.js

```diff
@@ -43,6 +43,12 @@ class ShardingManager extends EventEmitter {
     const stats = fs.statSync(this.file);
     if (!stats.isFile()) throw new Error('CLIENT_INVALID_OPTION', 'File', 'a file');
 
+    /**
+     * Enables the ability to rebalance the Shards.
+     * @type {boolean}
+     */
+    this.autoShards = options.totalShards === 'auto';
```

Comment:
why does this exist?  you don't use it anywhere.

Reply:
A few versions back I did, I forgot to remove it thank you.